cmd/collect/work_test.go

```diff
@@ -0,0 +1,95 @@
+package main
```

Comment:
Nice. I don't know why I didn't think of testing workers this way. This is good. Thank you!